### ISSUE TITLE
Config file for format string of parameters

### ISSUE THREAD

**Issue by Jammy2211** [rhayes777/PyAutoFit#33]
When we print parameters, we use a format string to display them, e.g. {:.2f}, {.4e}. Currently, there are methods which use format strings to do this, but the format string is input by the user or assumes a default value.

For every parameter, there are different ways we want to disply them. For exaple, some parameters may crrespond too very small numbers like 0.0000001, so a format string {:.2e} is appropriates. Others may warrant a :.2f.

Basically, we just need a config file to accompany label.ini which lists these format strings.

**Comment by rhayes777** [comment]
Can you give an example of where a printed parameter needs to be customised?

**Comment by Jammy2211** [comment]
In PyAutoLens, an example is the following summarize method. Basically, I can imagine that anything we can print / write to a results file needs its own format string (and label). This means that we made need new things in label.ini which previously werent there (for example, the mass of something).


```
    @dim.convert_units_to_input_units
    def summarize_in_units(self, radii, prefix='', whitespace=80,
                           unit_length='arcsec', unit_mass='solMass',
                           redshift_profile=None, redshift_source=None, cosmology=cosmo.Planck15, **kwargs):

        summary = super().summarize_in_units(
            radii=radii, prefix='', unit_length=unit_length, unit_mass=unit_mass,
            redshift_profile=redshift_profile, redshift_source=redshift_source, cosmology=cosmology, kwargs=kwargs)

        einstein_radius = self.einstein_radius_in_units(unit_length=unit_length,
                                             redshift_profile=redshift_profile, cosmology=cosmology, kwargs=kwargs)

        summary += [af_text_util.label_value_and_unit_string(label=prefix+'einstein_radius', value=einstein_radius,
                                                        unit=unit_length, whitespace=whitespace, format_str='{:.2f}')]

        einstein_mass = self.einstein_mass_in_units(unit_mass=unit_mass, redshift_profile=redshift_profile,
                                                    redshift_source=redshift_source, cosmology=cosmology, kwargs=kwargs)

        summary += [af_text_util.label_value_and_unit_string(label=prefix+'einstein_mass', value=einstein_mass,
                                                        unit=unit_mass, whitespace=whitespace, format_str='{:.4e}')]

        for radius in radii:

            mass = self.mass_within_circle_in_units(unit_mass=unit_mass, radius=radius, redshift_profile=redshift_profile,
                                                    redshift_source=redshift_source, cosmology=cosmology,
                                                    kwargs=kwargs)

            summary += [text_util.within_radius_label_value_and_unit_string(
                prefix=prefix+'mass', radius=radius, unit_length=unit_length, value=mass,
                unit_value=unit_mass, whitespace=whitespace, format_str_value='{:.4e}')]

        return summary
```

**Comment by rhayes777** [comment]
Configuration can now be set like so:

[format]
redshift = {:.2f}
luminosity = {:.4e}
einstein_radius = {:.2f}
einstein_mass = {:.4e}
mass = {:.4e}

In the case of labels that include _within_, the prefix is taken to be the config key. e.g. luminosity_within_500_pc is interpreted as luminosity.
